composer.json

```diff
@@ -9,12 +9,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.0.*|5.1.*"
+        "illuminate/support": "4.2.*|5.0.*|5.1.*|5.2.*"
```

Comment:
All of this scope stuff changed from nightmare hellscape to pleasant meadow full of flowers in 5.2, so I'd recommend following Landlord. I'll leave this package here for legacy people but it won't get any updates, and it's marked as abandoned in packagist.